example/cz_app/dialogs/main.xml

```diff
@@ -5,8 +5,10 @@
     <autogenerate type="back" propagate="true" on="false"/>
 -->
     <import>E_CZ_welcome.xml</import>
+<!--
```

Comment:
Tests use dialog from [tests folder](watson-assistant-workbench/tests/data/dialog/) (quite imperfect and will be changed). I commented out the generated part of dialog because with it, it is not possible to run only xml2json conversion whn you start playing with the xml.. But I can revert it.

Reply:
I will let it in your hands, but my recommendation is to run always all tests that were so far added... and improve the system to suit your needs in the future commits.

Reply:
Commenting this out does not affect tests as we have them now, I'll let it as it is. Further improvements can be done by #92 . But thank you for pointing this out!